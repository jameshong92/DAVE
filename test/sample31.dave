// test file output
int main() {
    str path = "test.txt";
    tbl test = load(path);
    path = "new.txt";
    save(test, path);
    tbl test2 = load(path);
    pritn(test2);
}