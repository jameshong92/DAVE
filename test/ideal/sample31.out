height	name	gender	weight	
178.1	tom	male	68	
195.2	jacky	male	81	
175.3	jay	male	77	
60.4	janice	female	48	
174.5	sammy	female	69	

